attrs/utils_test.go

```diff
@@ -0,0 +1,20 @@
+package attrs_test
```

Comment:
This can just be `package attrs` then you can remove the `attrs.` from usage in this file.

Reply:
Fixed!